reana_client/api/client.py

```diff
@@ -166,7 +184,12 @@ def get_workflow_status(workflow, access_token):
 
 
 def create_workflow(reana_specification, name, access_token):
-    """Create a workflow."""
+    """Create a workflow.
+
+    :param reana_specification: An object representing the REANA specification of the workflow.
```

Comment:
```suggestion
    :param reana_specification: a dictionary representing the REANA specification of the workflow.
```
What do you think? (note also the lowercase `a` instead of `An`)